flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -364,6 +364,7 @@ function check_logs_for_errors {
       | grep -v "HeapDumpOnOutOfMemoryError" \
       | grep -v "error_prone_annotations" \
       | grep -v "Error sending fetch request" \
+      | grep -v "WARN  akka.remote.ReliableDeliverySupervisor" \
```

Comment:
@LinyuYao1021 Do we still need this change?

Reply:
This is to avoid failure under this scenario:

```
2021-03-11T23:25:41.9106886Z Mar 11 23:25:41 2021-03-11 23:25:39,736 WARN  akka.remote.ReliableDeliverySupervisor                       [] - Association with remote system [akka.tcp://flink-metrics@10.1.0.4:37981] has failed, address is now gated for [50] ms. Reason: [Disassociated] 
2021-03-11T23:25:41.9108202Z Mar 11 23:25:41 2021-03-11 23:25:39,747 WARN  akka.remote.ReliableDeliverySupervisor                       [] - Association with remote system [akka.tcp://flink@10.1.0.4:37839] has failed, address is now gated for [50] ms. Reason: [Disassociated] 
2021-03-11T23:25:41.9109453Z Mar 11 23:25:41 2021-03-11 23:25:40,010 WARN  akka.remote.transport.netty.NettyTransport                   [] - Remote connection to [null] failed with java.net.ConnectException: Connection refused: /10.1.0.4:37839
2021-03-11T23:25:41.9111511Z Mar 11 23:25:41 2021-03-11 23:25:40,010 WARN  akka.remote.ReliableDeliverySupervisor                       [] - Association with remote system [akka.tcp://flink@10.1.0.4:***@10.1.0.4:37839]] Caused by: [java.net.ConnectException: Connection refused: /10.1.0.4:37839]
```

We already ignore `grep -v "WARN  akka.remote.transport.netty.NettyTransport"`

Reply:
> @LinyuYao1021 Do we still need this change?

Yes, it skips the `akka` exception check for logs.